src/libraries/Microsoft.Extensions.Http/src/PACKAGE.md

```diff
@@ -0,0 +1,81 @@
+## About
+
+<!-- A description of the package and where one can find more documentation -->
+
+[Microsoft.Extensions.Http](https://www.nuget.org/packages/Microsoft.Extensions.Http) package provides `AddHttpClient` extension methods for `IServiceCollection`, `IHttpClientFactory` interface and its default implementation. This provides the ability to set up named `HttpClient` configurations in a DI container and later retrieve them via an injected `IHttpClientFactory` instance.
```

Comment:
Nit: I wonder if you could come up with a more general description, that focuses more on the primary reason(s) of using `IHttpClientFactory` than the specific `AddHttpClient` API-s?